style/c.md

```diff
@@ -55,15 +55,15 @@ Atop your .c and .h files should be a comment that summarize what your program (
 // Says hello to the world
 ```
 
-You may also wish you to include your name, the date, and any other relevant information in your comments atop your .c and .h files:
+You may also wish to include your name, the date, and any other relevant information in your comments atop your .c and .h files:
```

Comment:
Let's remove this sentence altogether as well as the sample comment below it (with John Harvard's name), which we no longer recommend.